src/index.js

```diff
@@ -149,7 +149,7 @@ class UglifyJsPlugin {
   }
 
   apply(compiler) {
-    const requestShortener = new RequestShortener(compiler.context);
+    const requestShortener = new RequestShortener(compiler.context || process.cwd());
```

Comment:
@michael-ciniawsky - Agree here. We should get context without using `process. || window || w/e`